### DISCUSSION TITLE
Deleted page not removed from "recent pages" in admin dashboard

### DISCUSSION THREAD

**Discussion by dannyblaker** [requarks/wiki#5726] [Error / Bug Report]
**behviour:** after deleting page from admin, page is still listed under `recent pages` in dashboard at `http://localhost/a/dashboard` (see image 1 below)

**expected behviour:** after deleting a page, it should no longer be listed under `recent pages` as: 

- a) clicking the deleted page results in an error (see image 2 below)
- b) it causes confusion as the admin is unsure which pages still exist under "recent" - which begins to render the feature unusuable as number of deleted pages increases.

**suggested solution:** When a user deletes a page, update or refresh of the list of pages under `recent pages` in the dashboard view  `http://localhost/a/dashboard`

**origin:** My company is trialing wiki.js - the issue above came to attention when an wiki.js admin tried to open a page under recent pages, got the error (image 2 below), and thought there was something wrong with wiki.js and raised a ticket with IT dep. (that would be me :D)  

Many Thanks! :)

image 1
![image](https://user-images.githubusercontent.com/19355009/194783435-a8b8dafb-2874-40cd-ab31-741bd9a88eac.png)

image 2
![image](https://user-images.githubusercontent.com/19355009/194783522-629bb671-3453-4fdd-9cdf-d22b3bbb2de1.png)

**Comment by NGPixel** [comment]
Most likely browser cache? Try loading the page in another browser or in incognito mode.
